frontend/test/metabase/scenarios/question/notebook.cy.spec.js

```diff
@@ -97,6 +98,25 @@ describe("scenarios > question > notebook", () => {
       cy.contains("Showing 1 row");
     });
 
+    it.skip("should show correct column title with foreign keys (Issue #11452)", () => {
+      // Join tables with foreign keys
+      openOrdersTable();
+      cy.get(".Icon-notebook").click();
+      cy.findByText("Join data").click();
+      cy.findByText("Reviews").click();
+      cy.findByText("Product ID").click();
+      popover().within(() => {
+        cy.findByText("Product ID").click();
+      });
+
+      // Column titles should not include 'Product'
+      cy.get(".QueryBuilder .Icon-sum").click();
+      cy.findByText("Pick a column to group by").click();
+      cy.get(".ReactVirtualized__List").within(() => {
```

Comment:
Could this use `popover`? There might be multiple virtualized lists on the page.

Reply:
Yeah, that also works :)